students/WWhite/lesson04/trigrams.py

```diff
@@ -0,0 +1,48 @@
+import string, random
+
+file = open('sherlock_small.txt', 'r')
```

Comment:
It's a best practice to use a `with` statement when working with files which will automatically close the file when you exit the `with` block